tests/installation/bootloader_svirt.pm

```diff
@@ -82,11 +82,13 @@ sub run {
     if (check_var('BOOTFROM', 'c')) {
         $boot_device = 'hd';
     }
-    elsif (check_var('BOOTFROM', 'd')) {
+    elsif (check_var('BOOTFROM', 'd') || (get_var('ISO') && !get_var('BOOT_HDD_IMAGE'))) {
```

Comment:
You are right, that extra check is not needed...  http://phobos.suse.de/tests/1747963

```suggestion
    elsif (check_var('BOOTFROM', 'd') || get_var('ISO')) {
```